README.md

```diff
@@ -22,12 +22,15 @@ Or install it yourself as:
 
 ## Usage
 
-Cryptozoologist provides two main functions:
+Cryptozoologist provides four main functions:
 
 * `Cryptozoologist.random`: returns a string separated by a delimiter (_Note_: aliased as `Cryptozoologist.generate` for backwards compatibility with anything below version 3)
 * `Cryptozoologist.lorem(sentence_count)`: returns `sentence_count` number of sentences, separated by punctuation randomly selected from `["!", ".", "?"]`
 * `Cryptozoologist.street_address`: returns a string formatted to US street address standards (house number and street name) using the animal dictionaries (_Note_: ignores animal subdictionary exclusions)
 
+* `Cryptozoologist.state`: returns a string of a state name combined with a random word from other internal dictionaries.
```

Comment:
i think there's an extra blank line between this bullet and the previous one that shouldn't be there.